lib/sharedFunctions.bash

```diff
@@ -239,7 +239,7 @@ function trackAndTracePostFromFile() {
 				2>&1 | tee -a "${TRACE_FAILED}" \
 				&& log4Bash 'ERROR' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Failed to POST track&trace info using action ${_action} for entityTypeId=${_entityTypeId} from file=${_file} to https://${MOLGENISSERVER}/plugin/importwizard/importFile" \
 				2>&1 | tee -a "${TRACE_FAILED}" \
-				&& return
+				&& return 1
```

Comment:
Return 1 levert een exit state van niet nul. In combinatie met ```set -e``` in het script dat deze aanroept klappert dat script er dan uit en gaat niet meer verder; dat is denk ik niet wat we willen. Het betekend dat als de trace voor de email notificaties gaan en de trace klapt eruit met een exit state van niet nul, dat er dan ook geen email meer gaat komen....

Reply:
de trackAndTrace word aangeroepen met een if, we doen zelf aan de error handling